docs/installation/autoware/source-installation.md

```diff
@@ -2,7 +2,17 @@
 
 ## Prerequisites
 
-- [Ubuntu 20.04](https://releases.ubuntu.com/20.04/)
+- OS
+
+  - [Ubuntu 20.04](https://releases.ubuntu.com/20.04/) (**active development**)
+  - [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) (**will be supported in 2022**)
+
+- ROS
+
+  - ROS 2 Galactic (**active development**)
+  - ROS 2 Humble (**will be supported in 2022**)  
+    Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system dependencies.
```

Comment:
Could you wait for #139 and apply the same style? :pray: 
```suggestion
    For the system dependencies, refer to [REP-2000](https://www.ros.org/reps/rep-2000.html).
```

Reply:
Done.

Reply:
Have you pushed your change?

Reply:
Yes, already pushed.